UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.API.UnitTests/Storage/ExchangeSetStorageProviderTests.cs

```diff
@@ -16,13 +16,13 @@ public class ExchangeSetStorageProviderTests
     {
         private ExchangeSetStorageProvider service;
         private IOptions<EssFulfilmentStorageConfiguration> fakeStorageConfig;
-        private IAzureBlobStorageClient fakeAzureBlobStorageClient;
+        private IAzureBlobStorageService fakeAzureBlobStorageClient;
```

Comment:
I can see we have updated the Interface to use the Service instead of the Client but the field is still called a Client.

Is this correct?

If we are Faking a Service it might be less confusing to call it a Service?

Reply:
Yes did those changes. Right that needs to be renamed.

Reply:
did changes accordingly

Reply:
yes did changes accordingly.